entries/o/openclassrooms.com.json

```diff
@@ -0,0 +1,11 @@
+{ 
+  "OpenClassrooms": { 
+    "domain": "openclassrooms.com", 
+    "img": "openclassrooms.com.png", 
```

Comment:
```suggestion
```
SVG files are preferred over PNG files. Please remove the PNG and replace it with this SVG file:
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><rect width="64" height="64" fill="#7451eb" ry="3.2"/><path fill="#fff" d="M52.3 31.8a25.5 25.5 0 0 1-40.6 20.4l8.6-10a12 12 0 1 0 0-20.3l-8.6-10.1c4.3-3 9.5-5 15-5a25 25 0 0 1 25.6 25Z"/></svg>
```